addons/point_of_sale/models/res_company.py

```diff
@@ -41,6 +41,10 @@ def _load_pos_data_fields(self, config_id):
             'account_fiscal_country_id',
         ]
 
+    def _get_default_account_price_include(self):
+        # OVERRIDES account
+        return 'tax_included'
```

Comment:
This is not the case in US for instance, where people are used to see the price w/o taxes even in stores.

Reply:
It's in the specs: POS = tax_included as default.
The US can put it in the chart template, now or in a later moment.

Reply:
> It's in the specs

I know. POs make mistakes :)

Reply:
You have better proposals? To just remove this?

Reply:
I think it's been discussed in a talk with FP 😖

Reply:
Freak, I'm deleting this, I find no simple way to get outta this.

- If I put it as a function, it gets evaluated as soon as the module is read, so when you install it was `included` even if you were going to install PoS.
When you create a new company, it will be `tax_included` because the function will be re-evaluated

- Now, with the lambda, the value gets evaluated when it's read and not on install-time, so if I also install PoS, all the tests that rely on the default being `tax_excluded` just break.

We can think of a right way to do this later.